src/core/server/http/integration_tests/router.test.ts

```diff
@@ -304,126 +304,235 @@ describe('Options', () => {
   });
 
   describe('timeout', () => {
-    it('should timeout if configured with a small timeout value for a POST', async () => {
-      const { server: innerServer, createRouter } = await server.setup(setupDeps);
-      const router = createRouter('/');
+    describe('payload', () => {
```

Comment:
Do you all think we should have independent tests for the payload timeout's for POST/PUT/DELETE? Or can we rely on the unit tests for this?

Reply:
It bootstraps the whole Http Service, so a test for POST request suffices IMO.